src/annotation.js

```diff
@@ -598,6 +605,8 @@ var pointAnnotation = function (args) {
 };
 inherit(pointAnnotation, annotation);
 
+registerAnnotation('point', pointAnnotation, {point: true});
```

Comment:
@manthey Just as a side note,  heatmap annotations look like the old-school MS Paint spray can tool. I've seen papers where this is used to allow users to generate such distributions over vernacular geographic entities (e.g.  downtown).  There might actually be some useful applications of an annotation type like this (especially in medical imaging?)